content/en/apps/guides/database/muting_in_dashboards.md

```diff
@@ -0,0 +1,78 @@
+When a contact gets muted, two of [many things](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#muting) happen:
```

Comment:
Use rel and relref link shortcodes. See guide [here](https://gohugo.io/content-management/cross-references/).